app/assets/javascripts/admin/modules/edition-form.js

```diff
@@ -37,5 +38,32 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     })
   }
 
+  EditionForm.prototype.toggleWorldNewsStoryVisibility = function () {
```

Comment:
I think this would be clearer if it was name `setupWorldNewsStoryVisibilityToggle` or similar